### ISSUE TITLE
Download function for students?

### ISSUE THREAD

**Issue by orkit** [dsv-su/play#21]
A download for students. Should this be implemented? If so the feature should probably only download the video files (not the thumb or the posters).

**Comment by booski** [comment]
Yes, a feature to download a complete presentation for local playback should be implemented, similar to how mediasite currently does it (zip everything together with an html file containing the player).

This feture should be toggleable per presentation and default to being off (no download). We also need to think about where to expose the functionality - a button in the player controls? As part of the presentation card?

Either way, the feature can wait until after initial launch.

**Comment by orkit** [comment]
Alright. A small button in the presentation card would be a simple solution.
